### ISSUE TITLE
model.compile doesn't accept 'momentum' as an optimizer

### ISSUE THREAD

**Issue by ToonTalk** [tensorflow/tfjs#929]
To get help from the community, check out our [Google group](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs).

#### TensorFlow.js version
{tfjs-core: "0.13.11", tfjs-layers: "0.8.5", tfjs-converter: "0.6.7", tfjs: "0.13.5"}
#### Browser version
Version 70.0.3538.110 (Official Build) (64-bit)
#### Describe the problem or feature request
When specifying 'momentum' as the optimizer in model.compile one gets:
tfjs:2 Uncaught Error: Unknown Optimizer momentum
    at new t (tfjs:2)
    at getOptimizer (tfjs:2)
    at t.compile (tfjs:2)
    at t.compile (tfjs:2)
    at sample.html:16
Documentation says any tf.train.optimizer should work: "optimizer (string|tf.train.Optimizer) An instance of tf.train.Optimizer or a string name for an Optimizer." All the other optimizers listed at https://js.tensorflow.org/api/latest/#class:train.Optimizer can be specified by string.
Note that tf.train.momentum(.01, .9) will work.
#### Code to reproduce the bug / link to feature request
model.compile({loss: 'meanSquaredError', optimizer: 'momentum'});

**Comment by caisq** [comment]
@TooTalk Thanks for filing this issue. Can you provide some details on the momentum optimizer that you want to be able to specify as a string? It is not supported in keras or tensorflow.keras either.

**Comment by ToonTalk** [comment]
I'm developing material to be used by high school students. What I need is a reasonable default value for the momentum parameter. All the other optimizers can be chosen by using a string. Most of them also have additional parameters but they are optional (other than the learning rate). I'm unclear why the second argument in https://js.tensorflow.org/api/latest/#train.momentum isn't optional and given a default value.

**Comment by ToonTalk** [comment]
If there are good reasons to keep things the way they are then at least the documentation should be clearer since "An instance of tf.train.Optimizer or a string name for an Optimizer" leads one to think that all the optimizers listed at https://js.tensorflow.org/api/latest/#class:train.Optimizer can be specified by a string.

**Comment by rthadur** [comment]
Closing this due to lack of activity, feel to reopen. Thank you
